public/partials/user/detail.html

```diff
@@ -4,7 +4,7 @@
 			<fieldset>
 				<legend>Edit user {{user.userName}}</legend>
 				<div class="pull-right">
-					<img ng-src="http://srv-ind-scrat:8060/avatar/{{user.userName}}?s=40" />
```

Comment:
As the username is filled in this form, the {{user.userName}} is updated dynamically.
So while changing the user name, you can directly see the results for the avatar.
In the pull request, that changes as the url is only set in the backend, changes are only visible after save. 

Do you have a specific use case for changing the way of setting the avatar link?
In case you want to support other location in future, it is better to put the ?s= also in saved url and give the img tags a size. In that case the avatars can be requested with s=40 in all cases, can it can be resized to 32x32.
Using ?=s was introduced to request the images as small as possible (network load) and still have good pictures. 

In case you put a size on the image tags and get the avatar from a different location, then it will still be displayed with the right size.